tests/listeners/actions/test_sample_action.py

```diff
@@ -0,0 +1,46 @@
+import logging
+from unittest.mock import Mock
+
+from slack_bolt import Ack
+from slack_sdk import WebClient
+
+from listeners.actions.sample_action import sample_action_callback
+
+test_logger = logging.getLogger(__name__)
+
+
+class TestSampleAction:
+    def setup_method(self):
+        self.fake_ack = Mock(Ack)
+        self.fake_body = {"view": {"id": "test_id", "hash": "156772938.1827394"}}
+
+        self.fake_client = Mock(WebClient)
+        self.fake_client.views_update = Mock(WebClient.views_update)
+
+    def test_sample_action_callback(self):
+        sample_action_callback(
+            ack=self.fake_ack,
+            body=self.fake_body,
+            client=self.fake_client,
+            logger=test_logger,
+        )
+
+        self.fake_ack.assert_called_once()
+
+        self.fake_client.views_update.assert_called_once()
+        kwargs = self.fake_client.views_update.call_args.kwargs
```

Comment:
nit: views_update_args instead of kwargs may be easier to understand what the test code is doing but it's up to you!

Reply:
Could you elaborate on this 🤔 I tried using `.views_update.call_args.args` but since the values are passed as keyword arguments,  `.views_update.call_args.args` is empty

Also explored using `assert_called_once_with` but this method does not allow "fuzzy" argument matching like 
```python
self.fake_client.views_update.assert_called_once_with(
    view_id=self.fake_body["view"]["id"],
    hash=self.fake_body["view"]["hash"], 
    view=lambda v: v is not None  # this is not supported only exact matches are allowed 
)
```